markdown/blog/asyncapi-website-migration.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Migration of AsyncAPI Website to Next.js v14 and TypeScript"
+date: 2024-06-02T12:00:00+05:30
+type: Engineering
+tags:
+  - Project Status
+  - Website
+  - Migration
+  - TypeScript
+  - Next.js v14
+cover: /img/posts/asyncapi-website-migration.webp
+authors:
+  - name: Ashmit JaiSarita Gupta
+    photo: /img/avatars/ashmit-jaisarita-gupta.webp
+    link: https://ashmit.dev/
+    byline: Modelina Website Maintainer
+excerpt: 'Migration journey of the AsyncAPI Website, features introduced, and future plans.'
+---
```

Comment:
Add a tag as `featured: true` to feature this blog at top.

Reply:
Done